From 4ce1116a988fa2b0172ec96797a0a81ff4899c35 Mon Sep 17 00:00:00 2001
From: Rex Lu <rex.lu@mediatek.com>
Date: Wed, 29 Nov 2023 13:56:52 +0800
Subject: [PATCH 2018/2032] mtk: wifi: mt76: mt7996: Remove wed_stop during L1
 SER

Align logan L1 SER flow. During L1 SER, didn't need to close wed interrupt.

Signed-off-by: Rex Lu <rex.lu@mediatek.com>
---
 mt7996/mac.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/mt7996/mac.c b/mt7996/mac.c
index 912ae650..142e9372 100644
--- a/mt7996/mac.c
+++ b/mt7996/mac.c
@@ -1954,12 +1954,6 @@ void mt7996_mac_reset_work(struct work_struct *work)
 	dev_info(dev->mt76.dev,"\n%s L1 SER recovery start.",
 		 wiphy_name(dev->mt76.hw->wiphy));
 
-	if (mtk_wed_device_active(&dev->mt76.mmio.wed_hif2))
-		mtk_wed_device_stop(&dev->mt76.mmio.wed_hif2);
-
-	if (mtk_wed_device_active(&dev->mt76.mmio.wed))
-		mtk_wed_device_stop(&dev->mt76.mmio.wed);
-
 	ieee80211_stop_queues(mt76_hw(dev));
 	if (phy2)
 		ieee80211_stop_queues(phy2->mt76->hw);
-- 
2.18.0

